pkg/bindings/connection.go

```diff
@@ -91,9 +91,12 @@ func NewConnectionWithIdentity(ctx context.Context, uri string, identity string,
 	var connection Connection
 	switch _url.Scheme {
 	case "ssh":
-		port, err := strconv.Atoi(_url.Port())
-		if err != nil {
-			return nil, err
+		port := 22
```

Comment:
You could read /etc/services for this, but I think this is fine.
LGTM

Reply:
hesitant since there are a bunch of different ssh related lines in there it seems but I can do this if we want